pkg/kev/project.go

```diff
@@ -53,6 +120,24 @@ func WithSkaffold(c bool) Options {
 	}
 }
 
+func WithManifestFormat(c string) Options {
```

Comment:
It'd be great to have the exported functions documented. Here and other places in this PR. 🙏

Reply:
They're documented - I think I missed this one.

Reply:
Done